brainscore_vision/models/cvt_cvt_w24_384_in22k_finetuned_in1k_4/requirements.txt

```diff
@@ -0,0 +1,4 @@
+numpy
+torch
+transformers==4.19.0
+PIL
```

Comment:
@pellegreene PIL is no longer supported for Python 3 and instead has been replaced by [Pillow](https://pillow.readthedocs.io/en/stable/), same as https://github.com/brain-score/vision/pull/946